src/targets/chrome/docker.js

```diff
@@ -5,6 +5,7 @@ const { execSync } = require('child_process');
 const execa = require('execa');
 const waitOn = require('wait-on');
 const CDP = require('chrome-remote-interface');
+const fsExtra = require('fs-extra');
```

Comment:
Can you change this import to `fs` just to be consistent?

Reply:
Whoops! I was thinking I should open a different file where it was used to maintain style, but got lazy. Thanks for the callout!